bind/motor-mobile/object_builder.go

```diff
@@ -1,34 +1,20 @@
 package motor
 
 import (
-	"errors"
 	"fmt"
 
 	_ "golang.org/x/mobile/bind"
 )
 
-/*
-#include <stdlib.h>
-*/
-import "C"
-
-func NewObjectBuilder(n, d *C.char) error {
+func NewObjectBuilder(name, schemaDid string) error {
```

Comment:
Is it possible to bind an `ObjectBuilder` interface whose methods are exported to the target language? This could probably shoulder off a lot of code duplication. [See this article](https://medium.com/@matryer/tutorial-calling-go-code-from-swift-on-ios-and-vice-versa-with-gomobile-7925620c17a4)

Reply:
That makes sense. I will look into this in a subsequent PR. Filed #535